src/handlers/track_table.rs

```diff
@@ -123,37 +119,30 @@ fn handle_recommended_tracks(app: &mut App) {
 }
 
 fn jump_to_end(app: &mut App) {
-    match &app.track_table.context {
-        Some(context) => match context {
-            TrackTableContext::MyPlaylists => {
-                if let (Some(playlists), Some(selected_playlist_index)) =
-                    (&app.playlists, &app.selected_playlist_index)
+    if let Some(context) = &app.track_table.context {
+        if let TrackTableContext::MyPlaylists = context {
```

Comment:
Could this `if let` just become a normal `if context == TrackTableContext::MyPlaylists`?